source/configuration_manual/dict/index.rst

```diff
@@ -0,0 +1,351 @@
+:ref:`memcached-binary <dict-memcached_binary>` Memcached (Binary protocol)
+:ref:`proxy <dict-proxy>`                       Proxy
+:ref:`redis <dict-redis>`                       Redis
+:ref:`sql <dict-sql>`                           SQL
+=============================================== ===========================
+
+
+.. _dict-file:
+
+Flat Files
+----------
+
+.. code-block:: none
+
+  file:<path>
+
+The file will simply contain all the keys that are used. Not very efficient
+for large databases, but good for small ones such as a single user's quota.
+
+
+.. _dict-fs:
+
+Filesystem (lib-fs wrapper)
+---------------------------
+
+.. versionadded:: v2.2.11
+
+.. code-block:: none
+
+  fs:<driver>:<driver args>
+
+This is a wrapper for lib-fs, which most importantly has the ``posix``
+backend. So using:
+
+.. code-block:: none
+
+  fs:posix:prefix=/var/lib/dovecot/dict
+
+Would create a separate file under ``/var/lib/dovecot/dict`` for each key.
+
+
+.. _dict-ldap:
+
+LDAP
+----
+
+.. versionadded:: v2.2.24
+
+LDAP support is very similar to :ref:`dict-sql` support, but there is no write
+support.
+
+See :ref:`authentication-ldap`.
+
+Configuration
+^^^^^^^^^^^^^
+
+.. code-block:: none
+
+  dict {
+    somedict = ldap:/path/to/dovecot-ldap-dict.conf.ext
+  }
+
+Parameters in .ext file:
+
++--------------+---------------------------------------------------------------+
+| Parameter    | Description                                                   |
++==============+===============================================================+
+| ``bind_dn``  | DN or upn to use for binding. *OPTIONAL*                      |
++--------------+---------------------------------------------------------------+
+| ``debug``    | Enable debug. ``0`` = off (DEFAULT), ``1`` = on. *OPTIONAL*   |
++--------------+---------------------------------------------------------------+
+| ``password`` | Password to use, only SIMPLE auth is supported at the moment. |
+|              | *OPTIONAL*                                                    |
++--------------+---------------------------------------------------------------+
+| ``timeout``  | How long to wait for reply, in seconds. Default = 30 seconds. |
+|              | *OPTIONAL*                                                    |
++--------------+---------------------------------------------------------------+
+| ``tls``      | Use TLS?                                                      |
+|              | ``yes``: Require either ldaps or successful start TLS;        |
+|              | ``try``: Send start TLS if necessary (DEFAULT);               |
+|              | ``no``:  Do not send start TLS.                               |
+|              | *OPTIONAL*                                                    |
++--------------+---------------------------------------------------------------+
+| ``uri``      | LDAP connection URI as expected by OpenLDAP. **REQUIRED**     |
```

Comment:
I put the required one first. And I refactored the table to put Required as a separate column.  Just because we haven't done it before doesn't mean it's not a good idea :)  So we can discuss more.

Reply:
I guess. Although it would make sense to have similar column in the other tables in the same page. Of course, would need to find out which of them are required. I think it's always just the "host", and "dbname" for sql.

Reply:
Oh, I see there's a default value for everything else. So it's only the LDAP URI which is required ..

Reply:
True that no other driver has required parameters, but makes sense to keep table design consistent across the page.  Additionally, I added/clarified a few more default values.